tools/meteor-services/config.js

```diff
@@ -107,7 +107,9 @@ export function getPackageStorage(options) {
 }
 
 export function getIsopacketRoot() {
-  if (inCheckout()) {
+  if (process.env.METEOR_WAREHOUSE_DIR)
```

Comment:
Commit talks about `METEOR_WAREHOUSE_METADATA_DIR` (which still does not exist), but the code is about `METEOR_WAREHOUSE_DIR`. Should be consistent.

Reply:
Fixed